test/round_test.rb

```diff
@@ -66,4 +67,32 @@ def test_record_guess_edge_cases
     assert_equal 0, round.number_correct
   end
 
+  def test_dynamic_sleep_ability
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau")
+    card_2 = Card.new("Approximately how many miles are in one astronomical unit?", "93,000,000")
+    card_3 = Card.new("Is Taylor Swift the best artist of our era?", "yes")
+    deck = Deck.new([card_1, card_2, card_3])
+    round = Round.new(deck)
+
+    assert_equal 1, round.delay_output(1)
+    assert_equal 2, round.delay_output(2)
+  end
+
+  def test_check_guess
```

Comment:
Whenever there are multiple scenarios based on a condition, we want to test both of them.